### ISSUE TITLE
.venv folders not being ignored

### ISSUE THREAD

**Issue by GBrachetta** [tg44/heptapod#6]
Hi!

I open a new issue to keep things a bit more organised.

The app is working really a lot more reliably, but `.venv` folders are not being ignored after running.

Others, non dot folders are working great and I don't have now `node_modules` folders in my Time Machine backups, but `.venv` are still there.

They don't show up either when I run `heptapod tm ls`, so perhaps there's an issue with the dot starting the name of the folder?

I hope I'm clear!

Note: The `.venv` folders **show up in the dryrun and the log file**, but are not added to the Time Machine Ignore.

**Comment by tg44** [comment]
This is a tricky one :)

The problem is basically this; https://apple.stackexchange.com/a/55497

The tools I use under the hood;
 - `tmutil addexclusion ~/asd/.env` adds ~/asd/.env to the time machine exclusion (if the folder/file exists)
 - `tmutil removeexclusion ~/asd/.env`removes...
 - `tmutil isexcluded ~/asd/.env`if the line starts with [Excluded] it is excluded (I use it for per file check before I add/remove)
 - `mdfind com_apple_backup_excludeItem = 'com.apple.backupd'` lists the files/folders with the `com_apple_backup_excludeItem` label, but this works from some kind of index which doesn't contains some hidden folders :(

**Comment by GBrachetta** [comment]
That's fine, as soon as it's excluded. It took a full Time Machine backup to finally disappear from the backup file. It didn't the first time.

Good to know the command to remove exclusions: experimenting with Heptapod I created a lot of unnecessary folders and I would like to remove them now. For this then the command would be `tmutil removeexclusion ~/test/.env`, correct? I don't think there is an Heptapod command to do that yet.

But maybe a very nice solution (I don't know if it's possible) would be to have a `heptapod clean` command with which the app checks if folders previously added still exist and, if not, remove them.

By the way, I watched Arrival because of this app. 😀
